src/main.cpp

```diff
@@ -2247,6 +2277,12 @@ void vcRenderWindow(vcState *pProgramState)
     vcCamera_SwapMapMode(pProgramState);
 #endif
 
+  if (ImGui::IsKeyPressed(SDL_SCANCODE_F8))
+  {
+    pProgramState->screenshot.taking = true;
+    vcModals_TakeScreenshot(pProgramState, pProgramState->screenshot.outputName, vcMaxPathLength);
```

Comment:
It would be better to move the configuration to settings so that pressing the button takes it immediately (or as immediately as possible) without the modal popping up.

This will prevent things moving that the user wanted to capture.

An additional setting for "Show Screenshots" to open the display modal optionally would be useful.

Reply:
This block never opened the modal, but either way I've mostly changed to move settings into the settings dock and removed it from the modal, and I suppose I'll hook it up for show screenshots to bring up the classic image viewer